docs/architecture/design-proposals/OCI-registries.md

```diff
@@ -0,0 +1,7 @@
+# OCI support Helm charts in Kubeapps
+
+The goal of this document is to evaluate what’s needed to support OCI registries in Kubeapps to host Helm charts. We will identify blockers (if any) and propose an approach for a POC.
+
+## Design overview
+
+You can find the [design doc here](https://docs.google.com/document/d/1-6cKxOsW6K5u3lK7Om2zQeVYVPxzHT6dVwej5wy3_9A/edit).
```

Comment:
Copy-n-pasto :P It's currently linking to the multicluster design doc.

Reply:
:facepalm: